### PR TITLE
Add docs for forgotPassword

### PR DESCRIPTION
**Pull request by dvassallo**

### CODE REVIEW

File: src/pages/docs_sdk_forgot-password.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        // email with temporary password sent
+      })
+      .catch((e) => console.error(e))</code>
+  </pre>
+
+  <h3 id="params">Parameters</h3>
+
+  <ul>
+    <li>
+      <span class="field">username</span> [string | Len: 1-50] - The username for the account to send a temporary password to.
```
Comment (j-berman) [inline src/pages/docs_sdk_forgot-password.html]: Need to change this elsewhere too, but the max length we enforce server-side is actually 100

File: src/pages/docs_sdk_forgot-password.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    </li>
+  </ul>
+
+  <h3 id="errors">Errors</h3>
+
+  <ul>
+    <li>UsernameCannotBeBlank</li>
```
Comment (j-berman) [inline src/pages/docs_sdk_forgot-password.html]: Also UsernameTooLong

File: src/pages/docs_sdk_forgot-password.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    <li>UsernameMustBeString</li>
+    <li>AppIdNotSet</li>
+    <li>AppIdNotValid</li>
+    <li>UserNotFound</li>
+    <li>UserEmailNotFound</li>
+    <li>ServiceUnavailable</li>
```
Comment (j-berman) [inline src/pages/docs_sdk_forgot-password.html]: KeyNotFound
KeyInvalid

File: src/pages/docs_sdk_forgot-password.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
...t only when the user has provided an email address during <a href="/docs/sdk/sign-up/">signUp</a> or <a href="/docs/sdk/update-user/">updateUser</a>, and when the user has previously signed in with the <span class="field">rememberMe</span> option set to 'local'. In this case, the user will have the encryption key saved in the browser's local storage, and the user will be able to regain access to the account by getting a temporary password via email. The user must still have access to a previously used device in order to be able to reset the password. Therefore, if you want to allow your users to reset their password, make sure to set <span class="field">rememberMe</span> to 'local' during <a href="/docs/sdk/sign-up/">signUp</a> and <a href="/docs/sdk/sign-in/">signIn</a>, and make sure you require an email address during <a href="/docs/sdk/sign-up/">signUp</a> and <a href="/docs/sdk/update-user/">updateUser</a>. We still recommend that you inform your users that since their data is end-to-end encrypted, they should take care to store their password in a safe place, such as a password manager. Recovery will not be possible if the user loses access to all the previously used devices.
```
Comment (j-berman) [inline src/pages/docs_sdk_forgot-password.html]: Maybe we want to add a similar note to rememberMe in signUp and signIn? Some might not see this note here in forgotPassword before making their decision on rememberMe

File: src/pages/docs_sdk_forgot-password.html [hunk 5]
```diff
@@ -0,0 +1,44 @@
+<div class="section">
+  <h2><a href="/docs/">Docs</a> : <a href="/docs/sdk/">SDK</a> : forgotPassword</h2>
+
+  <p><span class="font-semibold">forgotPassword</span> lets you send a temporary password to the user's email address. This API will return a promise that gets resolved once the email with the temporary password has been sent.</p>
```
Comment (j-berman) [inline src/pages/docs_sdk_forgot-password.html]: Just commenting here so we don't forget

We also mentioned adding a temporary password flag in the response to signIn()

File: src/pages/docs_sdk_forgot-password.html (hunk 2 above)
Comment (j-berman) [reply src/pages/docs_sdk_forgot-password.html]: + ParamsMustBeObject

**Comment by dvassallo** [comment]
Done in this commit: https://github.com/encrypted-dev/userbase-homepage/commit/783a4996565f808fbba0d38bf6d8218dd8f0a39c

**Pull request closed without merging**